llvm/include/llvm/CodeGen/MIRYamlMapping.h

```diff
@@ -206,6 +207,8 @@ template <> struct MappingTraits<VirtualRegisterDefinition> {
     YamlIO.mapRequired("class", Reg.Class);
     YamlIO.mapOptional("preferred-register", Reg.PreferredRegister,
                        StringValue()); // Don't print out when it's empty.
+    YamlIO.mapOptional("flags", Reg.RegisterFlags,
+                       std::vector<FlowStringValue>());
```

Comment:
Can this be just {}

Reply:
DefaultT and T are separate, so the type can't be deduced.